IOMC/ParticleGuns/src/CloseByParticleGunProducer.cc

```diff
@@ -25,9 +25,12 @@ CloseByParticleGunProducer::CloseByParticleGunProducer(const ParameterSet& pset)
   ParameterSet defpset;
   ParameterSet pgun_params = pset.getParameter<ParameterSet>("PGunParameters");
 
+  fControlledByEta = pgun_params.getParameter<bool>("ControlledByEta");
   fEnMax = pgun_params.getParameter<double>("EnMax");
   fEnMin = pgun_params.getParameter<double>("EnMin");
   fMaxEnSpread = pgun_params.getParameter<bool>("MaxEnSpread");
+  fEtaMax = pgun_params.getParameter<double>("MaxEta");
+  fEtaMin = pgun_params.getParameter<double>("MinEta");
   fRMax = pgun_params.getParameter<double>("RMax");
   fRMin = pgun_params.getParameter<double>("RMin");
```

Comment:
```suggestion
  if (fControlledByEta) {
    fEtaMax = pgun_params.getParameter<double>("EtaMax");
    fEtaMin = pgun_params.getParameter<double>("EtaMin");
    if (fEtaMax <= fEtaMin)
      LogError("CloseByParticleGunProducer") << " Fix EtaMin and EtaMax values in the configuration";
  } else {
    fRMax = pgun_params.getParameter<double>("RMax");
    fRMin = pgun_params.getParameter<double>("RMin");
    if (fRMax <= fRMin)
      LogError("CloseByParticleGunProducer") << " Fix RMin and RMax values in the configuration";
  }
```
could avoid mistakes while configuring

Reply:
Please also remove the useless `<< endl` at the end of the two `LogDebug` statements in this code